src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -107,6 +107,33 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web config hostname', 'webapp', help="webapp name", completer=get_resource_name_completion_list('Microsoft.Web/sites'), id_part='name')
 register_cli_argument('appservice web config hostname', 'name', arg_type=name_arg_type, completer=get_hostname_completion_list, help="hostname assigned to the site, such as custom domains", id_part='child_name')
 
+database_types = ['SqlAzure', 'MySql']
+frequency_units = ['Hour', 'Day']
+
+register_cli_argument('appservice web config backup create', 'storage_account_url', help='SAS URL for the blob storage container')
```

Comment:
But what is a SAS URL?  It is just the SAS token, or the URL to the container in the storage account, WITH a SAS token appended?